usaspending_api/api_contracts/contracts/v2/references/total_budgetary_resources.md

```diff
@@ -11,35 +11,38 @@ This endpoint returns federal budgetary resources by fiscal year and fiscal peri
 
 + Parameters
 
-    + `fiscal_year`(optional, number)
+    + `fiscal_year` (optional, number)
         The fiscal year.
     + `fiscal_period` (optional, number)
-        The fiscal period. If this optional parameter is provided then `fiscal_year` is a required parameter
+        The fiscal period. If this optional parameter is provided then `fiscal_year` is a required parameter. If `fiscal_period` is provided without `fiscal_year`, a 400 error is returned.  Valid values: 1-12 (1 = October ... 12 = September). For retrieving quarterly data, provide the period which equals 'quarter * 3' (e.g. Q2 = P6). If neither paramater is provided, the entire available history will be returned.
```

Comment:
I think the period range should be 2-12 based on the reporting delay in a new fiscal year. You can verify there is no period 1 GTAS data by querying `gtas_sf133_balances`

Reply:
OK, changing it